src/System.Security.Cryptography.Cng/tests/TestData.cs

```diff
@@ -29,8 +29,15 @@ public static RSA CreateRsaCng(this RSAParameters rsaParameters)
             return rsa;
         }
 
-        public static RSAParameters TestRsaKeyPair;
+        public static ECDsa CreateEcCng(this ECParameters ecParameters)
+        {
+            ECDsa ec = new ECDsaCng();
+            ec.ImportParameters(ecParameters);
+            return ec;
+        }
 
+        public static RSAParameters TestRsaKeyPair;
+        public static ECParameters TestEcKeyPair;
```

Comment:
I'm not sure why we had something as under-specified as "TestRsaKeyPair", but can you help specify what this one is?  If nothing else, just ECParamsNistP256.